src/web/assets/cp/src/js/PreviewFileModal.js

```diff
@@ -73,13 +90,21 @@ Craft.PreviewFileModal = Garnish.Modal.extend(
       Craft.PreviewFileModal.openInstance = null;
       if (this.elementSelect) {
         this.elementSelect.focusItem(this.elementSelect.$focusedItem);
+      } else if (this.$triggerElement) {
```

Comment:
This could be set to an empty jQuery object, which would still evaluate to `true`, so should probably be:

```suggestion
      } else if (this.$triggerElement && this.$triggerElement.length) {
```